_drafts/2020-01-01-nyu-cs-transfer-perspective.md

```diff
@@ -16,13 +16,13 @@ programming skills and the recruiting scene is decent. I agree. But here are a f
 
 # Class Size
 
-There were 250 kids in my introductory courses. They were weed-out classes so the size made sense. However,
-there was usually a long line to get help on projects during office hours. As a new programmer, these time slots were crucial to
-understand the assignments and reinforces the concepts in lectures. I usually came early to avoid the crowd and was able to get guidance on difficult projects. Those who came 30 minutes later were not so fortunate. With such big class size, it left fewer opportunities to ask questions during class since the professors need to go through lectures in a timely manner.
+There were 250 kids in my introductory courses. They were weed-out classes so the size made sense. Coming from a high school with 350 people, it was certainly difficult for me to adjust. On the upside, the curriculum for my [class](https://www.cs.umd.edu/class/fall2018/cmsc131-020X040X/) was rigorous given I had a really great [professor](https://www.ratemyprofessors.com/ShowRatings.jsp?tid=313062).
+
+However, due to its size, there was usually a long line to get help on projects during office hours. As a new programmer, these time slots were crucial to understand the assignments and reinforces the concepts in the lectures. I usually came early to avoid the crowd and was able to get guidance on more difficult projects. Those who came 30 minutes later were not so fortunate. There were days that I saw 20 stressed students cramming into single room waiting for help from 1 TAs before the project was due. Not the greatest environment you wanted to be in. Overcrowdedness was also a problem during professor's office hours. Sometimes, there would be a line of people out the door waiting for help. Either you came early or came another time. Hence, it was hard to reach the lecturer to ask questions regarding the course materials. This could come at a disadvantage especially if you need recommendation letters and you haven't met with the professor for enough occasions. One of my friends also considered transferring but couldn't get a recommendation letter because he was told that he hadn't come to enough office hours. Additionally, the class size also left fewer opportunities to ask questions during class since the professors need to go through lectures in a timely manner.
```

Comment:
> As a new programmer, these time slots were crucial to understand the assignments and reinforces the concepts in the lectures.

This is a little awkward. How about

> As a new programmer, these time slots were crucial for me to understand the assignments and reinforce my comprehension of concepts from the lectures.

Reply:
1 TA not 1 TAs

Reply:
I generally prefer "In addition" to "Additionally" since imo "additionally" sounds a little colloquial or like slang. Not essential, up to you

Reply:
> Additionally, the *class* size also left fewer opportunities to ask questions during *class* since the professors need to go through lectures in a timely manner.

Having both "additionally" and "also" is redundant. Also notice how "class" repeats. Maybe we could figure out a way to avoid that repetition?